src/app/app.component.ts

```diff
@@ -117,6 +117,7 @@ export class AppComponent implements OnInit {
   strokeColor: string = '#FFFFFF';
   strokeWidth: number = 2;
   wrapTicks = false;
+  legendLabelStyle: LegendLabelStyle;
```

Comment:
Give it a default style to show a default value on the demo. Currently it's blank. 
<img width="364" alt="Screenshot 2024-01-09 at 6 54 56 PM" src="https://github.com/swimlane/ngx-charts/assets/47932418/00b3fd5c-0437-4d82-9625-b86f4dce3ab7">